libraries/AP_HAL_ChibiOS/hwdef/F4BY_F427/hwdef.dat

```diff
@@ -0,0 +1,169 @@
+# hw definition file for processing by chibios_hwdef.py
+# for F4BY v2.1.5 board description http://swift-flyer.com/?page_id=83
+
+# MCU class and specific type
+MCU STM32F4xx STM32F427xx
+
+# board ID for firmware load
+APJ_BOARD_ID 21
```

Comment:
Need to reserve an id in Tools/AP_Bootloader/board_list.txt and use its name in here

Reply:
ID is reserved. Tools/AP_Bootloader/board_types.txt How can I restart the tests now?